google-cloud-tools-plugin/testData/project/java.yaml

```diff
@@ -0,0 +1,15 @@
+runtime: java
+service: javaService
+env: flex
+
+handlers:
+- url: /.*
+  script: this field is required, but ignored
+  secure: always  # Require HTTPS
+
+runtime_config:  # Optional
+  jdk: openjdk8
+  server: jetty9
+
+manual_scaling:
+  instances: 1
```

Comment:
This was copied from https://cloud.google.com/appengine/docs/flexible/java/quickstart

Using an elaborate app.yaml already let me find a bug. And I guess it can stay there for future tests?